project_5.html

```diff
@@ -622,8 +620,7 @@ <h2 class="accordion-header">
 					<!--Pin to page?-->
 					<div id="schemCollapse3" class="accordion-collapse collapse" data-bs-parent="#schemAccordian">
 						<div class="accordion-body text-center">
-							<img src='assets/images/project_5/ops_23_project5_schemC.png' class="pj-schem"
-								style="max-width: 1000px;">
+							<img class="pj-img-fluid mx-auto" src='assets/images/project_5/ops_23_project5_schemC.png'>
```

Comment:
![image](https://github.com/uci-ieee/ops-webpages-2023-2024/assets/75911565/84bfe2c0-880f-4fcc-bbea-22ec57d1f2e6)

this one is slightly okay since it doesn't distort the schematic meaning much

![image](https://github.com/uci-ieee/ops-webpages-2023-2024/assets/75911565/b06bdd75-6f42-47df-bdd8-d616da8a2d04)

but overall these schematics are finicky on mobile